specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -414,7 +414,10 @@ model VMAttributes {
   @doc("Specifies whether the VMSize supporting local storage should be used to build Fleet or not.")
   localStorageSupport?: VMAttributeSupport;
 
-  @doc("The range of local storage in GB specified from Min to Max.")
+  @doc("""
+    LocalStorageSupport should be set to "Included" or "Required" to use this VMAttribute. 
+    If localStorageSupport is "Excluded", this VMAttribute can not be used." should be added to the description for localStorageInGiB.
```

Comment:
Can we please remove this? "**should be added to the description for localStorageInGiB**." from this?

Reply:
updated.